compute/instances/api-cli/creating-managing-instances-with-cliv2.mdx

```diff
@@ -0,0 +1,319 @@
+dates:
+  validation: 2021-08-05
+  posted: 2021-08-05
+---
+
+Scaleway's Instances provide you with resources to develop, test code and deploy your applications in no time. Instances are designed for modern workflows. They are with you from prototyping to large-scale deployment. You can create and manage your Instances from the [console](https://console.scaleway.com/login), via the API or via the Scaleway Command Line Interface.
+
+
+## Scaleway Command Line Interface Overview
+
+
+[The Scaleway Command Line Interface (CLI)](https://github.com/scaleway/scaleway-cli) allows you to pilot your Scaleway infrastructure directly from your terminal, providing a faster way to administer and monitor your resources. Scaleway CLI is easy to set up and use, and an essential tool for operating efficiently in your cloud environment. The CLI provides many functionalities, including the ability to create and manage Instances. 
+
+
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com/)
+  - You have [created an Instance](/compute/instances/how-to/create-an-instance/)
+  - You have configured your [SSH Key](/console/my-project/how-to/create-ssh-key/)
+  - You have generated your [API key](/console/my-project/how-to/generate-api-key/)
+  - You have downloaded and installed the [Scaleway CLI](https://github.com/scaleway/scaleway-cli#scaleway-cli-v2)
+  - You have configured the [Scaleway CLI](/compute/bare-metal-cloud/api-cli/bare-metal-with-cli/#-Configuring-the-CLI)
+
+</Message>
+
+
+
+## Configuring the CLI
+
+1. You must configure the Scaleway CLI on your machine. Open a terminal and type the following command:
+
+  `scw init`
+  
+  If you have already configured the CLI, you will see a message like the following:
+
+  ```
+  Current config is located at /path/to/config.yaml
+  access_key: <YOUR ACCESS KEY>
+  secret_key: <YOUR SECRET KEY>
+  default_organization_id: <YOUR ORGANIZATION ID>
+  default_project_id: <YOUR PROJECT ID>
+  default_region: <YOUR DEFAULT REGION, eg fr-par>
+  default_zone: <YOUR DEFAULT ZONE, eg fr-par-1>
+
+  Do you want to override the current config? (Y/n):  
+  ```
+
+2. Unless you want to change your configuration, you can type `n` and go directly to the [Creating an Instance](#creating-an-instance) step of this tutorial.
+
+  If you have not yet configured the CLI, you will be guided through the configuration process via a series of questions:
+
+  ```
+  Enter a valid secret-key or an email
+  ```
+
+3. Enter either the email address for your account (you will then be asked for your password), or your [API Secret Key](/console/my-project/how-to/create-ssh-key/):
+
+  ```
+  To improve this tool we rely on diagnostic and usage data.
+  Sending such data is optional and can be disabled at any time by running "scw config set send-telemetry=false".
+  Do you want to send usage statistics and diagnostics? (Y/n):
+  ```
+
+4. Type `y` or `n` and hit enter.
```

Comment:
here I suggest the same as we discussed for the k8s doc. I think you can remove the numbered steps.